MonoGame.Framework/Graphics/SpriteBatch.cs

```diff
@@ -51,9 +47,7 @@ public SpriteBatch (GraphicsDevice graphicsDevice)
 
 			this.GraphicsDevice = graphicsDevice;
 
-            // Use a custom SpriteEffect so we can control the transformation matrix
-            _spriteEffect = new Effect(graphicsDevice, EffectResource.SpriteEffect.Bytecode);
```

Comment:
This is on intentional. We create an Effect from  SpriteEffect.Bytecode to skip the SpriteEffect.OnApply(),
because we already calculate and apply _matrixTransform  inside Setup().